### ISSUE TITLE
Failed to minify when used with create-react-app

### ISSUE THREAD

**Issue by prontiol** [sinonjs/sinon#1830]
**Describe the bug**
We use sinon as a part of our application built with create-react-app
After upgrading to v5.1.0 the `npm run build` command failed with an error:
```
Failed to minify the code from this file:
./node_modules/sinon/pkg/sinon-esm.js:4824
Read more here: http://bit.ly/2tRViJ9
```

**To Reproduce**
1. Create an application with `create-react-app`
2. Install `sinon@5.1.0`
3. Run `npm run build`

**Expected behavior**
Build completes successfully

**Comment by fatso83** [comment]
See #1829. Once it's approved this should be good. Be aware that once Sinon 6 ships this will break your build again should you choose to upgrade to that (meaning you need to fix your own build to work with ES2015 modules).

**Comment by fatso83** [comment]
This should be closed now that Sinon 5.1.1 is out and 5.1.0 has been deprecated.
